### PR TITLE
making repo clean, moving scripts to a directory and utils to another

### PR DESCRIPTION
**Pull request by godmodegalactus**

### CODE REVIEW

File: scripts/configure_local.sh [hunk 1]
```diff
@@ -64,7 +61,14 @@ else
       "$validator_stake_account" \
     --ledger "$ledgerDir" \
     --cluster-type "development" \
-    $SPL_GENESIS_ARGS \
+    --bpf-program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA BPFLoader2111111111111111111111111111111111 $binDir/spl_token-3.5.0.so \
```
Comment (KirillLykov) [inline scripts/configure_local.sh:64]: I wonder why these args are in two places now (this file and also scripts/spl-genesis-args.sh)?

File: scripts/spl-genesis-args.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+--bpf-program TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA BPFLoader2111111111111111111111111111111111 ${binDir}/spl_token-3.5.0.so --bpf-program Memo1UhkJRfHyvLMcVucJwxXeuD728EqVDDwQDxFMNo BPFLoader1111111111111111111111111111111111 $binDir/spl_memo-1.0.0.so --bpf-program MemoSq4gqABAXKb96qnH8TysNcWxMyWCqXgDLGmfcHr BPFLoader2111111111111111111111111111111111 $binDir/spl_memo-3.0.0.so --bpf-program ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL BPFLoader2111111111111111111111111111111111 $binDir/spl_associated-token-account-1.1.1.so --bpf-program Feat1YXHhH6t1juaWF74WLcfv4XoNocjXA6sPWHNgAse BPFLoader2111111111111111111111111111111111 $binDir/spl_feature-proposal-1.0.0.so --bpf-program DGKy8w8RtRsWB48qHa4yCd3AeP5uv4m3Qn7LU8z93RWV BPFLoader2111111111111111111111111111111111 $binDir/mango.so --bpf-program 3WAiypER8fm6vHjUPRiigGifq6ueSY645aYGH5Jj14pU BPFLoader2111111111111111111111111111111111 $binDir/serum_dex.so --bpf-program EoUiQKGpM4jsdb5oRnYnuWMaE4Gcey72QjEBbFxhk23C BPFLoader2111111111111111111111111111111111 $binDir/pyth_mock.so
```
Comment (KirillLykov) [inline scripts/spl-genesis-args.sh:1]: looks like was moved without `git mv` so git thinks that it is a new file

**Review by KirillLykov — APPROVED** [review]
Looks good

File: scripts/spl-genesis-args.sh (hunk 2 above)
Comment (godmodegalactus) [reply scripts/spl-genesis-args.sh:1]: Yeeah this file is no longer necessary.

File: scripts/configure_local.sh (hunk 1 above)
Comment (godmodegalactus) [reply scripts/configure_local.sh:64]: I had some issues while getting the variables to work.
I will remove this file in my next commit.

**Pull request merged**